openmls/src/group/mls_group/proposal.rs

```diff
@@ -384,4 +386,51 @@ impl MlsGroup {
 
         Ok((mls_message, proposal_ref))
     }
+
+    /// Updates group context extensions
+    ///
+    /// Returns an error when the group does not support all the required capabilities
+    /// in the new `extensions`.
+    #[allow(clippy::type_complexity)]
+    pub fn update_group_context_extensions<Provider: OpenMlsProvider>(
+        &mut self,
+        provider: &Provider,
+        extensions: Extensions,
+        signer: &impl Signer,
+    ) -> Result<
+        (MlsMessageOut, Option<MlsMessageOut>, Option<GroupInfo>),
+        CreateGroupContextExtProposalError<Provider::StorageError>,
+    > {
+        self.is_operational()?;
+
+        // Create group context extension proposals
+        let inline_proposals = vec![Proposal::GroupContextExtensions(
+            GroupContextExtensionProposal { extensions },
```

Comment:
You can use the `new` function here. Then you don't need to make the `extension` member crate public.